test/functional/ex_cmds/cd_spec.lua

```diff
@@ -0,0 +1,18 @@
+-- Specs for
+-- :cd, :tcd, :lcd
+
+local helpers = require('test.functional.helpers')
+local nvim execute, eq, clear, eval, feed =
```

Comment:
You have missing comma here. So `execute` is `helpers.nvim`, not `helpers.execute`.